SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -408,6 +421,125 @@ void FSpatialGDKEditorToolbarModule::OnPropertyChanged(UObject* ObjectBeingModif
 	}
 }
 
+bool FSpatialGDKEditorToolbarModule::GenerateDefaultLaunchConfig(const FString& LaunchConfigPath) const
+{
+	FString Text;
+	TSharedRef< TJsonWriter<> > Writer = TJsonWriterFactory<>::Create(&Text);
+
+	// Populate json file for launch config
+	Writer->WriteObjectStart(); // Start of json
+	Writer->WriteValue(TEXT("template"), TEXT("small")); // Template section
+	Writer->WriteObjectStart(TEXT("world")); // World section begin
+	Writer->WriteObjectStart(TEXT("dimensions"));
+	Writer->WriteValue(TEXT("x_meters"), 2000);
+	Writer->WriteValue(TEXT("z_meters"), 2000);
+	Writer->WriteObjectEnd();
```

Comment:
My worry is that this is specific to our projects and it's not changeable by others without modifying source.

Can be ticketed and done later, but let's allow a documented user side callback to override this?

Reply:
We already have an option for specifying your custom launch config. This is only mean to be used for initial porting from single to multiworkers

Reply:
Or better yet, maybe this can be the default flow? We don't ship any launch configs with the GDK, and only generate them. User can change the generator through a callback if they want.

Reply:
```
This is only mean to be used for initial porting from single to multiworkers
```

But it will break instantly if the requirement (say, for size) is different.

Reply:
Yes I'll probably create a ticket for this. Initially I was considering adding full support for defining launch configs but chose against it after speaking with mike as when the ticket initially was created we only wanted to make it easy to have two servers out of the box without having to learn the details of the launch config.

That said, we could generate a sensible default and then allow users to build custom templates through the settings menu in Unreal. It may need some designing before implementing though.

Reply:
Added https://improbableio.atlassian.net/browse/UNR-1016